wiki/Building-on-Linux.md

```diff
@@ -7,59 +15,65 @@
 **Note**: these aren't the only things built at compile time, but merely what the end-user is expected to use.
 
 ## Preparing your system
+This project depends upon the following Make sure you install these before you build. If you get lost you can always check the [github workflow](https://github.com/input-leap/input-leap/blob/master/.github/workflows/builds.yml) to see how we build for your distro. Depending on your distrobution you may need to install `dev` or `devel` pacakges to get the files need to build aginst the dependencies.
```

Comment:
remove This project depends upon the following
pacakges

Reply:
Otherwise Linux section lgtm

Reply:
Should be good now thanks